MapboxNavigation/UIEdgeInsets.swift

```diff
@@ -8,6 +9,11 @@ public extension UIEdgeInsets {
                             bottom: left.bottom + right.bottom,
                             right: left.right + right.right )
     }
+    
+    static func >(lhs: UIEdgeInsets, rhs: UIEdgeInsets) -> Bool {
+        return (lhs.top + lhs.left + lhs.bottom + lhs.right)
```

Comment:
This is fine for now, but I would like to call out that this is a weird way to calculate `greaterThan.`

I would have expected `lhs.top > rhs.top && lhs.bottom > rhs.bottom...`